kani-driver/src/harness_runner.rs

```diff
@@ -55,7 +80,8 @@ impl<'pr> HarnessRunner<'_, 'pr> {
         let results = pool.install(|| -> Result<Vec<HarnessResult<'pr>>> {
             sorted_harnesses
                 .par_iter()
-                .map(|harness| -> Result<HarnessResult<'pr>> {
+                .enumerate() 
+                .map(|(idx,harness)| -> Result<HarnessResult<'pr>> {
```

Comment:
```suggestion
                .map(|(idx, harness)| -> Result<HarnessResult<'pr>> {
```
Run `./scripts/kani-fmt.sh` to format the code.